packages/esm-patient-orders-app/src/components/orders-details-table.component.tsx

```diff
@@ -439,7 +446,7 @@ function OrderBasketItemActions({
         id="discontinue"
         itemText={t('cancelOrder', 'Cancel Order')}
         onClick={handleCancelClick}
-        disabled={alreadyInBasket}
+        disabled={orderItem.fulfillerStatus === 'DECLINED'}
```

Comment:
Presumably we want to preserve the disabled status if the order is already in the basket? Also, I'm not sure I understand the reasoning here. Cancelling an order that was declined by the fulfiller seems like a valid action to take. The only reason to not allow cancelling an order is if the order has already been cancelled.

Reply:
I've verified the requirements for this and changed it to use the `action` attribute and it will be disabled when the value `DISCONTINUE`